content-scripts/inject/module.js

```diff
@@ -48,6 +48,17 @@ history.replaceState = function () {
   return returnValue;
 };
 
+const originalPushState = history.pushState;
```

Comment:
This is OK, but from my experience, going inside/outside the editor uses `replaceState`, not `pushState`. Any idea why you had to add this? Maybe Scratch changes behavior depending on the browser?

Reply:
1. add `console.log('replaceState', arguments);` and `console.log('pushState', arguments);` in module.js to log calls to those
2. open this in a fresh new tab: https://scratch.mit.edu/projects/422112922/
3. click see inside
4. observe that a pushState is logged 
![image](https://user-images.githubusercontent.com/33787854/99883003-03248a00-2bea-11eb-892d-2c23877da15b.png)
but every subsequent see inside/see project page will use replaceState for some reason
Happens in Chrome and Firefox